scripts/generate-dev-environment.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs')
+
+const ENV_FILE = '.env'
+
+const LOCAL_URL_MAP =`ADMINUSERS_URL=http://localhost:9700
```

Comment:
If we're running in an environment with actual maps, maybe we could actually use them to make it easier to edit/ not whitespace reliant?

It might look something like:
```js
const localEnvironment = {
  LOCAL_SPECIAL_VARS: 'local-special-values'
}

const tunnelEnrionment = {}

const dockerEnrionment = {}

const sharedEnvironment = {
  SHARED_SPECIAL_VARS: 'shared-special-values'
}

const environment = { ...environment, ...localEnvironment }

const out = Object.entries(environment)
  .map(([ key, value ]) => `${key}=${value}`)
  .join('\n')

fs.writeFileSync('.env', out)
```